validator/validator-main.protoascii

```diff
@@ -993,6 +993,26 @@ tags: {
     dispatch_key: NAME_VALUE_DISPATCH
   }
 }
+# AMP metadata, name=amp-vars-*
+# Allows advertisers to pass information from creatives
+# to be included in amp-ad-metadata.
+tags: {
+  html_format: AMP4ADS
+  tag_name: "META"
+  spec_name: "meta name=amp-vars-*"
+  mandatory_parent: "HEAD"
+  unique: true
+  attrs: {
+    name: "content"
+    mandatory: true
+  }
+  attrs: {
+    name: "name"
+    mandatory: true
+    value_regex: "^amp-vars-.+"
+    dispatch_key: NAME_VALUE_DISPATCH
```

Comment:
This won't work, since the value is a regex. You will need to drop the dispatch key, though it may cause some error-message quality issues with any `<meta>` errors giving error messages related to this one tagspec.

Reply:
This doesn't sound great. Is there any way to get better errors?

Reply:
The suggestion above would resolve this issue, but it's changing the syntax.

Reply:
Thanks for the info, decided to go with multiple tags. Removed `dispatch_key`.